packages/client/src/actions/frames.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  CreateFrameTypedDataQuery,
+  FrameVerifySignatureResult,
+  SignFrameActionMutation,
+  VerifyFrameSignatureQuery,
+} from '@lens-protocol/graphql';
+import type {
+  CreateFrameEip712TypedDataFragment,
+  CreateFrameTypedDataRequest,
+  FrameLensManagerSignatureResultFragment,
+  SignFrameActionRequest,
+  VerifyFrameSignatureRequest,
+} from '@lens-protocol/graphql';
+import type { ResultAsync } from '@lens-protocol/types';
+import type { AnyClient, SessionClient } from '../clients';
+import type { UnauthenticatedError, UnexpectedError } from '../errors';
+
+/**
+ * Create Frame action typed data to be signed by user wallet
+ *
+ * @param request - The request object
+ * @returns Typed data for Frame request
+ * @experimental This function might change in the future release
+ *
+ * @example
+ * ```ts
+ * const result = await createFrameTypedData(client, {
```

Comment:
We use this naming convention to re-inforce in the example what it can be used.
```suggestion
 * const result = await createFrameTypedData(anyClient, {
```